benchmarks/Microsoft.AspNetCore.Routing.Performance/Matchers/MatcherAzureBenchmark.cs

```diff
@@ -30,7 +30,7 @@ public void Setup()
             _samples = SampleRequests(EndpointCount, SampleCount);
 
             _baseline = (BarebonesMatcher)SetupMatcher(new BarebonesMatcherBuilder());
-            _dfa = SetupMatcher(new DfaMatcherBuilder());
+            _dfa = SetupMatcher(CreateDfaMatcherBuilder());
```

Comment:
A lot of the churn here is due to adding some services to the DFA matcher's constructor.